src/controllers/dynakube/oneagent/daemonset/labels.go

```diff
@@ -2,9 +2,14 @@ package daemonset
 
 import "github.com/Dynatrace/dynatrace-operator/src/kubeobjects"
 
-// buildLabels returns generic labels based on the name given for a Dynatrace OneAgent
```

Comment:
we should have more comments, not fewer 😄

Reply:
the best comment is no comment.
if code needs commenting, most of the time the code needs refactoring.
additionally, each comment needs maintenance the same way code needs maintenance. since comments are mostly overlooked, they tend to contain misinformation after some time. so in the end, a developer has to read the code again anyway.

Reply:
> the best comment is no comment.
I disagree. We should at least have a basic understanding of what each function does without having to look up all the different places it's used.
Instead of removing the comment because it's outdated, updating it when the functionality is changed is a better long term solution.

Reply:
the best comment is no comment == the best is when no comment is needed

complex functions that are not straight-forward should have some explanation
but `BuildLabels` in the `daemonset` package is rather straight forward, and the only thing I could comment is the actual list of labels that I'm creating which would be just repeating the code but in a different place, which is just asking to get outdated.

good comments answer the question "why are we doing this" and not the "what are we doing"

(if you have to ask "what are we doing" == the code is bad in general )